docs/wiki/s3-lifecycle/Architecture.md

```
@@ -0,0 +1,121 @@
+# S3 Lifecycle — Architecture
+
+High-level overview of the lifecycle worker. For implementation detail, see [`weed/s3api/s3lifecycle/DESIGN.md`](https://github.com/seaweedfs/seaweedfs/blob/master/weed/s3api/s3lifecycle/DESIGN.md).
+
+## At a glance
+
+The lifecycle worker runs as a scheduled job. Each invocation:
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)